docs/content/en/performing-operations.md

```diff
@@ -91,6 +91,38 @@ Then we can update our newly created **Post**:
   </code-block>
 </code-group>
 
+And if we want to use `PATCH`, we can easily do that using [config](/api/query-builder-methods#config).
+
+<code-group>
+  <code-block Label="Query" active>
+
+  ```js
+  const post = await Post.find(1)
+  
+  post.text = 'An updated text for our Post!'
+  
+  await post.config({ method: 'PATCH' }).save()
+  ```
```

Comment:
I have a suggestion. Create a alias. What do you think?

`await post.patch()` => `await post.config({ method: 'PATCH' }).save()`

Reply:
Great idea! :smiley:

Reply:
Done in 68a4675